modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/view_calendar_menus.jspf

```diff
@@ -680,6 +680,20 @@
 					date: event.date,
 				});
 			},
+			keydown: function (event) {
+				var instance = this;
+
+				var domEvent = event.domEvent;
+
+				if (
+					domEvent.keyCode === 13 &&
+					domEvent.target.hasClass('yui3-calendar-day-selected')
+				) {
+					<portlet:namespace />scheduler.setAttrs({
```

Comment:
This seems like an odd pattern, generating a namespace for a javascript object, where did you get this from?

Reply:
I know, I just duplicated the dateClick function when we press enter: https://github.com/liferay-frontend/liferay-portal/blob/55ef14d39c1a91575fd1597f6c9f10301b50d2fa/modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/view_calendar_menus.jspf#L678

Reply:
Ah gotcha. I have no idea how that really works but I'll trust it since we've already been doing it